Common/src/main/kotlin/net/horizonsend/ion/common/database/collections/PlayerData.kt

```diff
@@ -12,7 +12,8 @@ class PlayerData private constructor(
 	@BsonId val minecraftUUID: UUID,
 	var discordId: Long? = null,
 	var minecraftUsername: String? = null,
-	var achievements: MutableList<Achievement> = mutableListOf()
+	var achievements: MutableList<Achievement> = mutableListOf(),
+	var voteTimes: MutableMap<String, Long> = mutableMapOf("Minecraft-MP" to 0, "Planet-Minecraft" to 0, "Minecraft-Server-List" to 0)
```

Comment:
Don't pre-populate it with non-config determined values

Reply:
leftover from testing